src/trip-repo.js

```diff
@@ -3,51 +3,95 @@ import Trip from "./trip.js";
 class TripRepo {
   constructor(tripData, destinationData) {
     this.allTrips = tripData;
-    this.destinationData = destinationData;;
+    this.destinationData = destinationData;
   }
 
   compareDates(d1, d2) {
-    const date1 = new Date(d1)
+    const date1 = new Date(d1);
     const date2 = new Date(d2);
-
     return date1 > date2 ? true : false;
```

Comment:
Not related to this PR, but the `>` comparison translates to a `boolean`, so you can just write `return date1 > date2;`.

Reply:
Thank you!